qa/test_template.sh

```diff
@@ -26,28 +26,35 @@ if [[ $one_config_only = true ]]; then
     last_config_index=0
 fi
 
+prolog=${prolog-:}
+epilog=${epilog-:}
+
 for i in `seq 0 $last_config_index`;
 do
-    echo "Test run $i"
-    # install packages
-    inst=$($topdir/qa/setup_packages.py -i $i -u $pip_packages --cuda ${CUDA_VERSION})
-    if [ -n "$inst" ]
-    then
-      pip install $inst
-
-      # If we just installed tensorflow, we need to reinstall DALI TF plugin
-      if [[ "$inst" == *tensorflow-gpu* ]]; then
-        pip uninstall -y nvidia-dali-tf-plugin || true
-        pip install /opt/dali/nvidia-dali-tf-plugin*.tar.gz
-      fi
-    fi
-    # test code
-    test_body
-
-    # remove packages
-    remove=$($topdir/qa/setup_packages.py -r  -u $pip_packages --cuda ${CUDA_VERSION})
-    if [ -n "$remove" ]
-    then
-      pip uninstall -y $remove
-    fi
+    for i in $(seq 0 ${#prolog[@]}); do
```

Comment:
```suggestion
    for j in $(seq 0 ${#prolog[@]}); do
```
and adjust below. You shadow the loop index.

Reply:
Done